### ISSUE TITLE
more granular report on require_once when it comes to URLs and functions returning a url

### ISSUE THREAD

**Issue by david-binda** [Automattic/VIP-Coding-Standards#26]
```
require_once(wpcom_vip_theme_url('/my-theme/my-file.php', 'my-other-theme'));
```

and

```
require_once( 'http://example.com/my-file.php' );
```

Should complain not b/c of not using absolute URL, but for the sake of using external URLs which is not supported on the WordPress.com VIP plaform.

**Comment by GaryJones** [comment]
Which sniff currently reports these false positives?

**Comment by david-binda** [comment]
Currently the output is as follows:

```
$ cat test.php
<?php

require_once(wpcom_vip_theme_url('/my-theme/my-file.php', 'my-other-theme'));

require_once( 'http://example.com/my-file.php' );
$ phpcs --standard=WordPressVIPMinimum ./test.php -s

FILE: /home/wpcom/public_html/test.php
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AND 1 WARNING AFFECTING 2 LINES
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 3 | WARNING | File inclusion using custom constant (`wpcom_vip_theme_url`). Probably needs manual inspection. (WordPressVIPMinimum.Files.IncludingFile.IncludingFile)
 5 | ERROR   | Absolute include path must be used. Use `get_template_directory()`, `get_stylesheet_directory()` or `plugin_dir_path()`. (WordPressVIPMinimum.Files.IncludingFile.IncludingFile)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 49ms; Memory: 6Mb
```

While those cases should be caught, it should be b/c of requesting a remote file, while the code should really be only requiring local files.
